provisioner/components/k8s/setup_playbook/common_setup/.travis.yml

```diff
@@ -0,0 +1,29 @@
+---
+language: python
+python: "2.7"
```

Comment:
Is it generated a file ? python version should be > 3.6.

Reply:
Yes it's a generated file that gets generated when we create an ansible role

Reply:
Updated with python 3.6

Reply:
Please check if code works with python3.6 by removing python 2.x from the system.

Reply:
Yes working